### PR TITLE
Light editing of FAQ material

### PR DESCRIPTION
**Pull request by jasongrout**
Including setting expectations for jlab 1.0, links to *stable* docs, etc.

### CODE REVIEW

File: packages/faq-extension/faq.md [hunk 1]
```diff
@@ -117,6 +117,6 @@ extend the functionality of JupyterLab. Examples of possible extensions include:
 To start developing your own JupyterLab extension, please have a look
 at:
```
Comment (jzf2101) [inline packages/faq-extension/faq.md]: We should include the API documentation here

**Comment by jzf2101** [comment]
I'm generally ok with this but how much like the README should we move people ASAP to the docs?

File: packages/faq-extension/faq.md [hunk 2]
```diff
@@ -47,39 +47,39 @@ Notebook. Here are a few of them you may want to try out:
...
 * Stable and featureful enough for daily usage.
-* Most of the commonly used features in the classic notebook are
+* Most of the commonly used features in the classic Notebook are
   implemented.
-* Developer APIs that are approaching stability but still undergoing
+* Developer APIs that are approaching stability, but still undergoing
   significant changes.
 
-Early in 2018, we will release the 1.0 version of JupyterLab that will
+In 2018, we will release the 1.0 version of JupyterLab that will
```
Comment (jzf2101) [inline packages/faq-extension/faq.md]: I actually think we don't need to include this paragraph in the FAQ - this I think can stay in the blogpost and docs.

File: packages/faq-extension/faq.md [hunk 3]
```diff
@@ -47,39 +47,39 @@ Notebook. Here are a few of them you may want to try out:
...
 provide additional UI/UX improvements, features, and API stability. At
 that point, JupyterLab should be a full featured replacement for the
-classic notebook - and go far beyond its capabilities. Between now and then
+classic Notebook and go far beyond its capabilities. Between now and then,
 we will release a series of beta releases, all of which should be stable
 for daily usage.
 
 ### What will happen to the classic Jupyter Notebook?
 
 JupyterLab is intended to be a full replacement for the classic Jupyter
```
Comment (jzf2101) [inline packages/faq-extension/faq.md]: I would actually swap the points in the paragraph to emphasize the fact that we have the same server and notebook support and mention retirement of the classic notebook.

File: packages/faq-extension/faq.md [hunk 4]
```diff
@@ -47,39 +47,39 @@ Notebook. Here are a few of them you may want to try out:
 * Double click on CSV files to view them as a nicely formatted table.
 * Drag and drop notebook cells within a notebook or between notebooks.
 
-See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/latest/)
+See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/stable/)
```
Comment (jzf2101) [inline packages/faq-extension/faq.md]: I'd actually move the docs link up to general so we can get people to the docs ASAP

**Comment by jasongrout** [comment]
@jzf2101 - you've got a lot of great suggestions. Feel free to either edit the faq directly on this PR (click the "edit" pencil icon in the files changed view to add another commit on top of mine here), or submit a PR on top of this one. I don't really have strong opinions about what it says here. I was just trying to make sure it wasn't contradicting the docs and fixing the grammar. Please go ahead and make the changes you'd like to make.

**Comment by jasongrout** [comment]
> I think we should just move all of this to the docs and not have it here at all

I opened #3835 for this, targeting beta 2.

**Comment by jasongrout** [comment]
Oops, one more typo: "signifciant" -> "significant"

File: packages/faq-extension/faq.md [hunk 5]
```diff
@@ -1,122 +1,41 @@
...
-  connect one to a text file.
-* Right click on a markdown file and “Open with” a live markdown viewer.
-* Double click on CSV files to view them as a nicely formatted table.
-* Drag and drop notebook cells within a notebook or between notebooks.
-
-See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/latest/)
-for more detailed information about these and other features.
+* [How can you report a bug or provide feedback?](https://github.com/jupyterlab/jupyterlab/issues)
+* [How can you contribute?](https://github.com/jupyterlab/jupyterlab/blob/master/CONTRIBUTING.md)
+* [How can you extend or customize JupyterLab?](jupyterlab.readthedocs.io/en/stable/user/extensions.html)
```
Comment (jasongrout) [inline packages/faq-extension/faq.md]: Need `https://` in the link

File: packages/faq-extension/faq.md [hunk 6]
```diff
@@ -1,122 +1,41 @@
...
+
 ## General
 
-* [What is JupyterLab?](#What-is-JupyterLab?)
-* [What new features does JupyterLab offer?](#What-new-features-does-JupyterLab-offer?)
+* [What is JupyterLab?](http://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html)
 * [How stable is JupyterLab?](#How-stable-is-JupyterLab?)
 * [What will happen to the classic Jupyter Notebook?](#What-will-happen-to-the-classic-Jupyter-Notebook?)
-* [Where is the documentation for JupyterLab?](#Where-is-the-documentation-for-JupyterLab?)
+* [Where is the documentation for JupyterLab?](http://jupyterlab.readthedocs.io/en/stable/)
```
Comment (jasongrout) [inline packages/faq-extension/faq.md]: `https`

File: packages/faq-extension/faq.md [hunk 7]
```diff
@@ -1,122 +1,41 @@
+# Welcome to the Frequently Asked Questions (FAQ) for the JupyterLab Beta release series
+
 ## General
 
-* [What is JupyterLab?](#What-is-JupyterLab?)
-* [What new features does JupyterLab offer?](#What-new-features-does-JupyterLab-offer?)
+* [What is JupyterLab?](http://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html)
```
Comment (jasongrout) [inline packages/faq-extension/faq.md]: `https`

**Comment by jasongrout** [comment]
> Oops, one more typo: "signifciant" -> "significant"

Fixed.

**Comment by jasongrout** [comment]
@jzf2101 - looks good to me. Feel free to merge after tests pass.

**Pull request merged**